app/javascript/entry_points/orientation.js

```diff
@@ -1,4 +1,4 @@
-import Vue from 'our_vue'
```

Comment:
this is stuff from the other PR, right?

Reply:
Oops, yes sorry. Forgot to change the base. Fixed now.